xml/docu_styleguide.language.xml

```diff
@@ -405,6 +405,49 @@
    </para>
    </sect2>
 
+ <sect2 xml:id="sec-quotation">
+  <title>Quotations</title>
+  <para>
+  Use quotations to quote from sources, such as books.
+  In all other cases, do not use quotation marks:
+  </para>
+  <itemizedlist>
+  <listitem>
+  <para>
+  For computer input, computer output, and user
+  interface elements, use different markup.
+  See also <xref linkend="sec-command-line"/> and
```

Comment:
making link to docbook tag list https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-taglis-docbook